### PR TITLE
[DOCFIX]Update Job service/stressBench related doc and description

### PR DESCRIPTION
**Pull request by jja725**
### What changes are proposed in this pull request?

See title

### Why are the changes needed?

Please clarify why the changes are needed. For instance,
  1. If you propose a new API, clarify the use case for a new API.
  2. If you fix a bug, describe the bug.

### Does this PR introduce any user facing changes?

Please list the user-facing changes introduced by your change, including
  1. change in user-facing APIs
  2. addition or removal of property keys
  3. webui

### CODE REVIEW

File: docs/en/operation/StressBench.md [hunk 1]
```diff
@@ -263,3 +263,542 @@ Here is a sample usage demo, where we have one master and three workers:
...
+
+#### Testing
+The cluster testing is similar to single node testing except that
+- `--cluster` argument needs to be added to each operation so that the bench jobs will be submitted to job master, distributed to job workers, and executed by job workers.
+  Each job worker executes one bench job. Cluster throughput is calculated by aggregating the read throughput of each bench job.
+
+### Limitations
+- Master Stress Bench only supports testing self-generated test files.
+
+## Worker Stress Bench
```
Comment (jja725) [inline docs/en/operation/StressBench.md:639]: Doesn't get much information about worker stress bench and we are not using it currently.

File: docs/en/operation/StressBench.md [hunk 2]
```diff
@@ -263,3 +263,542 @@ Here is a sample usage demo, where we have one master and three workers:
...
+        <td>conf</td>
+        <td>{}</td>
+        <td>Any HDFS client configuration key=value. Can repeat to provide multiple configuration values.</td>
+    </tr>
+  
+</table>
+
+### Single node testing
+#### Prerequisite
+* A running Alluxio cluster with one master, and one worker. With property "alluxio.user.file.writetype.default" = MUST_CACHE
```
Comment (dbw9580) [inline docs/en/operation/StressBench.md:470]:
```suggestion
* A running Alluxio cluster with one master, and one worker, with property `alluxio.user.file.writetype.default` set to `MUST_CACHE`
```

File: docs/en/operation/StressBench.md [hunk 3]
```diff
@@ -263,3 +263,542 @@ Here is a sample usage demo, where we have one master and three workers:
...
+```console
+$ bin/alluxio runClass alluxio.stress.cli.StressMasterBench --operation ListDir --warmup 5s --duration 30s 
+```
+#### MaxThroughput test
+MaxThroughput test is the recommended way to test the master throughput for certain operation.
+```console
+$ bin/alluxio runClass alluxio.stress.cli.suit.MaxThroughput --operation CreateFile ...
+```
+
+For example, we are trying to get max throughput of CreateFile operation using hadoop compatible client.
```
Comment (dbw9580) [inline docs/en/operation/StressBench.md:620]:
```suggestion
For example, we are trying to get max throughput of `CreateFile` operations using the Hadoop compatible client.
```

File: docs/en/operation/StressBench.md [hunk 4]
```diff
@@ -263,3 +263,542 @@ Here is a sample usage demo, where we have one master and three workers:
...
+```console
+$ bin/alluxio runClass alluxio.stress.cli.suit.MaxThroughput --operation CreateFile --warmup 5s --duration 30s --client-type AlluxioHDFS
+```
+
+
+### Cluster testing
+#### Prerequisite
+- A running Alluxio cluster. Each worker node contains at least one worker, one job worker.
+- Ufs has enough space to store the test data if you are testing operation contains file creation.
```
Comment (dbw9580) [inline docs/en/operation/StressBench.md:629]:
```suggestion
- UFS has enough space to store the test data if you are testing operations that involve file creation.
```

File: docs/en/operation/StressBench.md [hunk 5]
```diff
@@ -263,3 +263,542 @@ Here is a sample usage demo, where we have one master and three workers:
...
+For example, 
+```console
+$ bin/alluxio runClass alluxio.stress.cliStressWorkerBench --operation ListDir --warmup 5s --duration 30s 
+```
+
+
+### Cluster testing
+#### Prerequisite
+- A running Alluxio cluster. Each worker node contains one worker, one job worker.
+- Ufs has enough space to store the test data if you are testing operation contains file creation.
```
Comment (dbw9580) [inline docs/en/operation/StressBench.md:725]: Same as above.

File: docs/en/operation/StressBench.md [hunk 6]
```diff
@@ -263,3 +263,542 @@ Here is a sample usage demo, where we have one master and three workers:
...
+The cluster testing is similar to single node testing except that
+- `--cluster` argument needs to be added to each operation so that the bench jobs will be submitted to job master, distributed to job workers, and executed by job workers.
+  Each job worker executes one bench job. Cluster throughput is calculated by aggregating the read throughput of each bench job.
+
+### Limitations
+- Worker Stress Bench only supports testing self-generated test files.
+
+## Ufs IO Bench
+
+The Ufs IO Bench is a tool to measure the IO throughput between the Alluxio cluster and the UFS.This test will measure the I/O throughput between Alluxio workers and the specified UFS path. Each worker will create concurrent clients to first generate test files of the specified size then read those files. The write/read I/O throughput will be measured in the process.
```
Comment (dbw9580) [inline docs/en/operation/StressBench.md:737]:
```suggestion
The UFS IO Bench is a tool to measure the IO throughput between the Alluxio cluster and the UFS. This test will measure the I/O throughput between Alluxio workers and the specified UFS path. Each worker will create concurrent clients to first generate test files of the specified size then read those files. The write/read I/O throughput will be measured in the process.
```

File: docs/en/operation/StressBench.md [hunk 7]
```diff
@@ -263,3 +263,542 @@ Here is a sample usage demo, where we have one master and three workers:
...
+#### Prerequisite
+* A running Alluxio cluster with one master, and one worker.
+
+#### Single test
+Run the test with the operation you want to test.
+```console
+$ bin/alluxio runClass alluxio.stress.cli.UfsIOBench --io-size 1G ...
+```
+
+For example,This invokes the I/O benchmark to HDFS in the Alluxio cluster which each thread is writing and then reading 512m of data
```
Comment (dbw9580) [inline docs/en/operation/StressBench.md:787]:
```suggestion
For example, this runs the UFS I/O benchmark in the Alluxio cluster, where each thread is writing and then reading 512MB of data from HDFS:
```

File: docs/en/operation/StressBench.md [hunk 8]
```diff
@@ -263,3 +263,542 @@ Here is a sample usage demo, where we have one master and three workers:
...
+```console
+$ bin/alluxio runUfsIOTest --path hdfs://<hdfs-address> --io-size 512m --threads 2 
+```
+
+
+### Cluster testing
+#### Prerequisite
+- A running Alluxio cluster. Each worker node contains at least one worker, one job worker.
+- Ufs has enough space to store the test data if you are testing operation contains file creation.
```
Comment (dbw9580) [inline docs/en/operation/StressBench.md:796]: Same as above.

**Review by dbw9580 — COMMENTED** [review]
Thanks for the detailed docs! Left some minor suggestions, didn't check the content.

**Comment by jja725** [comment]
@dbw9580 should address all the comments, PTAL

File: docs/en/operation/StressBench.md [hunk 9]
```diff
@@ -263,3 +263,532 @@ Here is a sample usage demo, where we have one master and three workers:
...
+    <tr>
+        <td>duration</td>
+        <td>30s</td>
+        <td>The length of time to run the benchmark. (1m, 10m, 60s, 10000ms, etc.)</td>
+    </tr>
+    <tr>
+        <td>read-same-file</td>
+        <td>false</td>
+        <td>If true, all threads read from the same file. Otherwise, each thread reads "
+          + "from its own file.</td>
```
Comment (dbw9580) [inline docs/en/operation/StressBench.md:448]: You don't need string concatenation within HTML. Just wrap the line with a newline and any number of white spaces, browsers take care of the rest.

**Review by dbw9580 — COMMENTED** [review]
LGTM. Thanks!

**Review by yuzhu — APPROVED** [review]

**Comment by yuzhu** [comment]
alluxio-bot, merge this please

**Comment by jja725** [comment]
@Xenorith This is ready for cherry pick

**Comment by Xenorith** [comment]
alluxio-bot, cherry-pick this to branch-2.7 please

**Pull request merged**